conf/app.routes

```diff
@@ -1,10 +1,13 @@
 # microservice specific routes
 
 ->         /hmrc-frontend                   hmrcfrontend.Routes
-GET        /logout                          uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers.SignOutController.logout
+GET        /logout                          uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers.ServiceController.logout
+GET        /keep-alive                      uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers.ServiceController.keepAlive
 GET        /assets/*file                    controllers.Assets.versioned(path = "/public", file: Asset)
 GET        /language/:lang                  uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers.LanguageSwitchController.switchToLanguage(lang: String)
 
+GET        /                                uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers.ServiceController.homePage
+
```

Comment:
Not needed, can be deleted as the `/appeal-start` will be the start page for the service.